server/src/main/java/org/elasticsearch/index/mapper/KeywordFieldMapper.java

```diff
@@ -278,6 +279,13 @@ private Builder(
                 m -> toType(m).normalizerName,
                 null
             ).acceptsNull();
+            this.normalizerSkipStoreOriginalValue = Parameter.boolParam(
+                "normalizer_skip_store_original_value",
+                false,
```

Comment:
isn't it ok to go from false -> true here? Although probably not ok the other way around.

Reply:
Yes, that would work. Going from false -> true would cause previously indexed documents to start returning their normalized values, and the original values stored in _ignored_source would be unused.

I'm reluctant to allow it though, as I think it might create some unnecessary confusion. It's simpler to reason about and to debug if the values are all stored the same way throughout the life of the index.

Reply:
Agreed, let's keep this mapping attribute immutable.